packages/extension/src/components/pages/Login/Login.tsx

```diff
@@ -154,9 +176,17 @@ export const Login: FC = () => {
           });
       }
     } else {
-      setPasswordError('Incorrect password');
+      if (currentAttempts >= maxAttempts) {
+        let TIMESTAMP = Date.now();
+        setDisableLogin(true);
+        saveInChromeLocalStorage('disabledLoginTimer', TIMESTAMP + 1000 * 60 * 15);
```

Comment:
You could put that calculation within the timestamp constant already

Reply:
done in : 1ac1bd390fd0d24cd765dafe475216c48b573ce8